### DISCUSSION TITLE
2007/12/27/%E5%9B%BE%E6%A0%87%E7%BC%96%E8%BE%91%E5%99%A8%E4%B8%8A%E7%9A%84%E9%BC%A0%E6%A0%87%E5%8F%8C%E5%87%BB%E6%8A%80%E5%B7%A7

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#198] [Announcements]
# 2007/12/27/%E5%9B%BE%E6%A0%87%E7%BC%96%E8%BE%91%E5%99%A8%E4%B8%8A%E7%9A%84%E9%BC%A0%E6%A0%87%E5%8F%8C%E5%87%BB%E6%8A%80%E5%B7%A7

图标是 LabVIEW 语言很有特色的一个东西，其它语言大概都不需要给它的函数设置一个图形标识。认真给每一个 VI 加上一个有意义的图标是非常必要的，它可以帮助程序人员快速理解一段程序的含义。一个风格良好的 VI，只要扫一眼它的程序框图，就可以了解他的功能了，而不必要像文本语言那样一行行把程序读下来。LabVIEW 程序比文本语言的程序可读性要高得多，当然这是指在编程风格良好的情况下进行比较。

https://blog.qizhen.xyz/2007/12/27/%E5%9B%BE%E6%A0%87%E7%BC%96%E8%BE%91%E5%99%A8%E4%B8%8A%E7%9A%84%E9%BC%A0%E6%A0%87%E5%8F%8C%E5%87%BB%E6%8A%80%E5%B7%A7

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2009年03月11日 17:31 编辑
这个是我自己发现的，不过后来看了LabVIEW在线帮助，上面也有介绍。

回复
帆
2009年02月25日 16:20 编辑
这个技巧在哪里发现的？自己摸索还是有文档记载？

回复
刘伟印
2008年08月6日 13:12 编辑
不错，作者很细心啊

回复
agilentvee
2008年01月2日 17:05 编辑
Small Fonts”，字号8。好象太小,最好是Arial  12号字体

回复
竹贤
2007年12月27日 20:07 编辑
尽管是小技巧，但以前从未知道。随手用来，确实方便。希望能够多介绍一些，我们也可有所收益。
